app/admin.py

```diff
@@ -41,6 +41,7 @@ def get(self):
         view_config_json = dict(
                 (name, encoder.encode(value))
                 for name, value in view_config.iteritems())
+        configsan = encoder.encode(view_config)
```

Comment:
This was temporary name and we planned to discuss a better name. How about "all_view_config_json"?